README.md

```diff
@@ -1,57 +1,67 @@
+for the CabinDAO visual identity. Topo Components is our library of pre-built 
+UI components using [Storybook][5].
 
 ## Contributing
 
 This project is open source and we would love for you to contribute to it. In
-the [CabinDAO  discord server](https://discord.gg/4G6XjsCjM3), there is a channel dedicated to discussing this
-product. Progress, milestones, and on-going initiatives are tracked [here in
-GitHub](https://github.com/CabinDAO/design-system/projects). There is also
-a [discussion board](https://github.com/CabinDAO/design-system/discussions) here
-on GitHub that acts as a good public-interface to the development of this
-product.
+the our [discord server][11], there is a channel dedicated to discussing this 
+product. Progress, milestones, and on-going initiatives are tracked
+[here in GitHub][6]. There is also a [discussion board][7] here on GitHub that 
+acts as a good public interface to the development of this product.
 
-If you would like to contribute, start by reading the [Contribution Guide](./CONTRIBUTING.md) and the [ Code of Conduct ](CONDUCT.md). Open issues
-can be found on this repository.
+If you would like to contribute, start by reading the [Contribution Guide][8] 
+and the [Code of Conduct][9]. Open issues can be found in this repository.
 
 ## Setup
 
-To get the application up and running locally, consult the [Setup Guide](./SETUP.md).
-
-## Cabin
-
-[Cabin](https://www.creatorcabins.com) is a digitally-native organization (often
-called a DAO) building a decentralized city by creators, for creators.  You can
-learn more about us by [visiting our website](https://www.creatorcabins.com) or
-[joining our Discord server](https://discord.gg/4G6XjsCjM3). Please consult our
-[Brand Guidelines](https://github.com/CabinDAO/design-system) on how to work
-within our visual identity. 
+1. Fork the Topo repository into your Github account.
+2. Clone the forked repo to your local system.
+3. Navigate to the newly created project directory.
+4. `yarn` or `npm i` will install the Topo package along with its dependencies.
+
+---
+
+**[Cabin](https://www.creatorcabins.com)** is a digitally-native organization 
+(often called a DAO) building a decentralized city by creators, for creators. 
+You can learn more about us by [visiting our website][12] or [joining our Discord server][11].
+Please consult our [Brand Guidelines][13] on how to work within our visual identity. 
+
+We look forward to creating with you!
+
+[1]: https://github.com/CabinDAO/design-system/issues/new "Issues"
+[2]: https://github.com/CabinDAO/design-system/discussions "Discussions"
+[3]: https://github.com/CabinDAO/design-system/blob/main/CONTRIBUTING.md "Contributing"
+[4]: https://stitches.dev/docs/tokens "Theme Tokens"
+[5]: https://storybook.js.org/docs/react/get-started/introduction "Storybook Documentation"
+[6]: https://github.com/CabinDAO/design-system/projects "Projects"
+[7]: https://github.com/CabinDAO/design-system/discussions "Discussions"
```

Comment:
Let's swap these out with clarity links:
- Topo bounties instead of projects: https://app.clarity.so/cabin/view/b06b6487-3625-41d7-8d6e-2f7a1fd32061
- We'll have a forum feature on Clarity soon in place of discussions, let's just direct to the discord channel for now